app/config/tstvhq/rooms.json

```diff
@@ -279,8 +276,8 @@
             "muteRoom": false,
             "hideVideo": false,
             "description": "Give the goats from Woodstock Farm Sanctuary a snack at woodstocksanctuary.org/",
-            "type": "CHATSTREAM",
-            "backgroundImage": "https://tstvhq.s3.amazonaws.com/backgrounds/LUPERCALIA/LUPERCALIA-BG-sleeping-porch.png",
+            "type": "IFRAME",
```

Comment:
Is there a reason we want `IFRAME` and not `CHATSTREAM` here?

Reply:
IFRAME because we want people to be able to talk in this room

Reply:
oic